src/Query/Result.php

```diff
@@ -43,7 +51,21 @@ public static function createFromArray(array $arrayResult): self
             return $hitFormated;
         }, $arrayResult['hits']['hits']) : [];
 
-        return new self($totalHits, $hits);
+        $aggregations = isset($arrayResult['aggregations']) ? array_map(function ($aggregation) {
```

Comment:
I think that the ternary condition makes it difficult to read this code block.

Maybe it could be rewrite this an `if` block ?

Reply:
indeed, will also refactor the $hits assignments, which suffers from the same problem